src/main/java/tlylt/haha/Task.java

```diff
@@ -0,0 +1,106 @@
+package tlylt.haha;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Abstract representation of a task.
+ */
+public abstract class Task {
+    protected final String description;
+    protected boolean isDone;
+    protected final String type;
```

Comment:
Perhaps for the variables description and type, you could be more specific?

Reply:
ERM, Perhaps the comments could have been done in one single review, following the guide? Now my mailbox is bombed 💣